deployment/k8s-config/scripts/harbor-backup.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+#https://github.com/restic/restic/releases/download/v0.12.1/restic_0.12.1_linux_amd64.bz2
+#https://restic.readthedocs.io/en/stable/index.html
+#https://github.com/stonezdj/harbor/blob/backup_restore/tools/harbor-backup.sh
+
+RESTIC_REPO='s3:repo-url'
+KEY_ID='<aws access key id>'
+ACCESS_KEY='<aws access key>'
+PASSWORD='<restic password>'
```

Comment:
Where are these credentials being stored?

Reply:
I have them stored locally on my work machine. Is there a better place to put these?

Reply:
I don't know... is there an internal git repo that ops uses for secrets?

Reply:
It seems we don't have a consistent system in place for secrets between ops people. Setting that up might be a story in itself. For now I have the secrets in an encrypted database stored locally.